src/EventTickets/Migrations/CreateEventTicketTypesTable.php

```diff
@@ -49,7 +49,7 @@ public function run() {
 			created_at DATETIME NOT NULL,
 			updated_at DATETIME NOT NULL,
 			PRIMARY KEY  (id)
-		) $charset";
+		) $charset ENGINE=InnoDB;";
```

Comment:
What's the purpose of adding this?

Reply:
On my machine where the default engine is MyISAM, many of the tests crash because of their interactions.

Indeed, the teardown cleanup function of legacy tests uses SQL transactions with `ROLLBACK` to undo any modification of the DB by the test.

But MyISAM does not manage transactions...

This is why I suggest explicit use of the InnoDB engine.

Reply:
@pulsovi thanks for the clarification